near-contract-standards/src/fungible_token/core_impl.rs

```diff
@@ -105,8 +105,10 @@ impl FungibleToken {
         let balance = self.internal_unwrap_balance_of(account_id);
         if let Some(new_balance) = balance.checked_add(amount) {
             self.accounts.insert(account_id, &new_balance);
-            self.total_supply =
-                self.total_supply.checked_add(amount).expect("Total supply overflow");
```

Comment:
Needs double checking, but I *think* `.expect` should be pretty cheap, as it doesn't involve formatting machinery.

Reply:
Seems like it does, looking at the [rust src](https://github.com/rust-lang/rust/blob/e8e1b32a7840c07f30c04b252c379a044a73902d/library/core/src/option.rs#L1618). Calls panic which I assume adds line information and also uses formatting syntax (is fmt not used when formatting a `&str`?)

Reply:
Ah, right, `.expect` is `#[track_caller]`, so it will include line info!